### ISSUE TITLE
Minor Request for the FMC: Message to "Reset MCP Altitude" X nm before reaching initial T/D

### ISSUE THREAD

**Issue by oMrSmith** [mSparks43/747-400#526]
Dear @mSparks43,

don't worry, I plan to calm down
after the next release :D .

- [x] FMC Message to "Reset MCP Altitude" before reaching initial T/D

**Comment by crazytimtimtim** [comment]
By "X" do you mean 10?

**Comment by mSparks43** [comment]
![image](https://user-images.githubusercontent.com/6773686/117083154-53307c80-ad44-11eb-81bc-b56d3eb30513.png)

**Comment by crazytimtimtim** [comment]
Looking into this, shouldn't

**Comment by crazytimtimtim** [comment]
Looking into this, shouldn't
https://github.com/mSparks43/747-400/blob/702b3d107716cfb7af7c52f000c97713036e6866/plugins/xtlua/scripts/B747.68.xt.fms/activepages/B744.fms.pages.vnav.lrc.lua#L39
be using ground speed, not true airspeed?

**Comment by gHashFlyer** [comment]
Good!  Thank you.  I will make the change.

**Comment by crazytimtimtim** [comment]
> Good! Thank you. I will make the change.

ok. May I also create a dataref for the estimated TOD time? I don't know if you want me messing in your code.

**Comment by mSparks43** [comment]
> ok. May I also fix this issue (#526) in your code?

this is already done, just pending a push once the flight testing of #516 and #523 is finished, T/D eta is on the progress page
![image](https://user-images.githubusercontent.com/6773686/117088125-da382180-ad51-11eb-8070-75e019e7ff43.png)

**Comment by crazytimtimtim** [comment]
> > ok. May I also fix this issue (#526) in your code?
> 
> this is already done, just pending a push once the flight testing of #516 and #523 is finished, T/D eta is on the progress page

You mean the warning is done or the T/D eta is done?

**Comment by mSparks43** [comment]
> > > ok. May I also fix this issue (#526) in your code?
> > 
> > 
> > this is already done, just pending a push once the flight testing of #516 and #523 is finished, T/D eta is on the progress page
> 
> You mean the warning is done or the T/D eta is done? I've pretty much finished the warning.

Both, the warning just needed B747DR_fmc_notifications[33] setting to 1 in the ap monitor

**Comment by mSparks43** [comment]
You could pick some of the others tho, there's only a couple of these actually with logic behind them atm
https://github.com/mSparks43/747-400/blob/master/plugins/xtlua/scripts/B747.68.xt.fms/B744.notifications.lua

There's an excel file knocking around somewhere with a lot of those conditions

**Comment by gHashFlyer** [comment]
sure that's fine by me

**Comment by crazytimtimtim** [comment]
> Both, the warning just needed B747DR_fmc_notifications[33] setting to 1 in the ap monitor

oh, I was going to use ```lua
fmsO["notify"]="RESET MCP ALTITUDE"```

**Comment by mSparks43** [comment]
> oh, I was going to use `fmsO["notify"]="RESET MCP ALTITUDE"`

that way doesn't work, because the notification doesn't depend on FMC page, FMC, and there is no easy way to register if it was cleared on another FMC.

**Comment by mSparks43** [comment]
obviously starting a line with # doesnt put it in the commit message
https://github.com/mSparks43/747-400/commit/27120432c64bcafec7583214864747484b2449a5

**Comment by oMrSmith** [comment]
> By "X" do you mean 10?

I wasn't sure what the measurement was, but it seems like you figured it out!
It work very well and the message automatically clears out when resetting the MCP ALT.
Is it supposed to make a "bing" sound, too?

**Comment by crazytimtimtim** [comment]
> Is it supposed to make a "bing" sound, too?

I believe so. I think it should also illuminate the yellow message warning on the side of the FMC.

**Comment by mSparks43** [comment]
> I believe so. I think it should also illuminate the yellow message warning on the side of the FMC.

plse make an issue for this that I can add to R3, it'll have to wait because there is no lighting for MSG on the FMCs
